toolkits/springernature/packages/springernature-header/view/header.hbs

```diff
@@ -0,0 +1,36 @@
+<header class="c-header">
+    <div class="c-header__inner">
+        <div class="c-header__content">
+            <div class="u-flex-wrap">
+                <a data-test="header-logo-link" data-component="back-submissions" href="{{backUrl}}" class="u-flex-wrap header-logo-link" title="Back to manuscript list">
+                    <img alt="Springer Nature" width="209" src="/images/header_logo_SN.svg" class="{{#if showJournalLogo}}with-journal-logo {{/if}}site-logo">
+                    {{#if showJournalLogo}}
+                        <span class="hide journal-title" data-component-journal-title>{{submission.journalName}}</span>
+                        {{#if submission.journalId}}
+                            <img alt="" height="45" src="//ca-branding.live.public.springernature.io/journals/{{submission.journalId}}/logo.svg" class="journal-logo" 
+                            data-component-journal-logo onerror="this.classList.add('hide');document.querySelector('[data-component-journal-title]').classList.remove('hide');">
+                        {{/if}}
+                    {{/if}}
+                </a>
+            </div>
+
+            {{#userDetails}}
```

Comment:
I wouldn't put this code in here, because if the `user-details` component changes (or different versions have different HTML) then this won't work. Just put a comment saying that the HTML from the `user-details` component _can_ go here.